### ISSUE TITLE
Can we remove required parameters for request body?

### ISSUE THREAD

**Issue by akira-19** [exoego/rspec-openapi#195]
When I generate a document, parameters for request body are set as required.
Is there any way to set the parameters optional?
I would like to remove the following required section.

```
requestBody:
        content:
          application/json:
            schema:
              type: object
              properties:
                param1:
                  type: boolean
                param2:
                  type: boolean
                param3:
                  type: boolean
              required:
              - param1
              - param2
              - param3
```

**Comment by exoego** [comment]
It is to add a test case that does not use optional parameters.

**Comment by akira-19** [comment]
@exoego Thank you. I want it to have no required field. I mean all parameters are optional.

```
it `...` do
  # use all parameters
end

it `...` do
  # use no parameters
end
```

I tried this, and it didn't work though what you showed worked.

**Comment by akira-19** [comment]
@exoego Your example works, but it doesn't work when I try to make all the parameters optional like below.

```
it 'returns 201' do
        params = {
          param1: true,
          param2: true,
          param3: true
        }
        patch api_path, params: params, headers: token
        expect(response.status).to eq 201
      end

      it 'returns 201' do
        params = {}
        patch api_path, params: params, headers: token
        expect(response.status).to eq 201
      end
```

**Comment by exoego** [comment]
@akira-19 This was fixed in #196 and landed in v0.16.0.
